drivers/volume/portworx/portworx.go

```diff
@@ -1352,29 +1329,17 @@ func (d *portworx) UpgradeDriver(endpointURL string, endpointVersion string) err
 // GetClusterPairingInfo returns cluster pair information
 func (d *portworx) GetClusterPairingInfo() (map[string]string, error) {
 	pairInfo := make(map[string]string)
-	pxNodes, err := d.schedOps.GetRemotePXNodes(remoteKubeConfigPath)
-	if err != nil {
-		logrus.Errorf("err retrieving remote px nodes: %v", err)
-		return nil, err
-	}
-	if len(pxNodes) == 0 {
-		return nil, fmt.Errorf("No PX Node found")
-	}
 
-	clusterMgr, err := d.getClusterManagerByAddress(pxNodes[0].Addresses[0])
+	resp, err := d.clusterPairManager.GetToken(d.getContext(""), &api.SdkClusterPairGetTokenRequest{})
 	if err != nil {
 		return nil, err
 	}
-	resp, err := clusterMgr.GetPairToken(false)
-	if err != nil {
-		return nil, err
-	}
-	logrus.Infof("Response for token: %v", resp.Token)
+	logrus.Infof("Response for token: %v", resp.Result.Token)
 
 	// file up cluster pair info
-	pairInfo[clusterIP] = pxNodes[0].Addresses[0]
-	pairInfo[tokenKey] = resp.Token
-	pairInfo[clusterPort] = strconv.Itoa(pxdRestPort)
+	//pairInfo[clusterIP] = pxNodes[0].Addresses[0]
+	pairInfo[tokenKey] = resp.Result.Token
+	//pairInfo[clusterPort] = strconv.Itoa(pxdRestPort)
```

Comment:
is it correct?

Reply:
yes, this token in sdk now comes inside the Result object